blocks_vertical/data.js

```diff
@@ -468,10 +468,10 @@ Blockly.Constants.Data.CUSTOM_CONTEXT_MENU_GET_VARIABLE_MIXIN = {
    */
   customContextMenu: function(options) {
     if (!this.isCollapsed()) {
-      var variablesList = this.workspace.variableList;
+      var variablesList = this.workspace.getAllVariables();
```

Comment:
This should probably get all variables of '' type and then a different mixin should be added to lists to get variables of that type.